source/manual/manage-assets.html.md.erb

```diff
@@ -19,6 +19,8 @@ For example, for `https://assets.publishing.service.gov.uk/government/uploads/up
 
 ## Remove an asset
 
+Some applications, such as Whitehall, might require setting a redirect, rather than deleting the asset. Look out for replacements, or unpublished state of the parent edition, in the publishing application, and set the `replacement_id` or `redirect_url` in asset manager accordingly.
```

Comment:
Could we expand this out into its own "Redirect an asset" section instead? And maybe add some blurb at the top of the doc, where we explain when to redirect and when to simply remove.

I'd love to see this section expanded out with code examples (e.g. how to set the `redirect_url`? Whether to set `replacement_id` or `redirect_url` - which is preferred? etc)

Reply:
There already is a redirect a section but I wanted to put it in the delete section as a reminder for people like me who totally forgot there are other states and just fixed all zendesk ticket assets with deletion 🙈

I can do an intro section with an explanation of the various states of assets, at least what I understand of it so far. And move this bit in that section, at the top of the file.

The [redirect section](https://github.com/alphagov/govuk-developer-docs/blob/f635b86f7833b73fa437ab6e55bfcaf2e138c2c6/source/manual/manage-assets.html.md.erb#L48) uses a bit of an odd syntax for the rake task. [Rake task ](https://github.com/alphagov/asset-manager/blob/bf1189b102a2c6d104fe5c9df601c3fa801d895d/lib/tasks/assets.rake#L15)seems alright. Never used it though.

Reply:
Sounds good - let me know if you want a hand with syntax etc.